src/PIL/Image.py

```diff
@@ -1949,6 +1955,57 @@ def save(self, fp, format=None, **params):
             if open_fp:
                 fp.close()
 
+    def _convert_mode(self, plugin, params):
```

Comment:
I'd prefer to have a function which takes a list of modes, rather than a plugin. It could be useful in other places, not only for saving.

Reply:
Okay, done.